syslogish/server.go

```diff
@@ -106,13 +106,9 @@ func (s *Server) receive() {
 		if err != nil {
 			log.Fatal("syslogish server read error", err)
 		}
-		rawMessage := strings.TrimSuffix(string(buf[:n]), "\n")
-		// We want to strip off all the leading syslog junk but there is no good way to do that.
-		// So we split the string on the leading "{" but we have to add it back so we can parse the log message into a map
-		logMessage := "{" + strings.SplitAfterN(rawMessage, "{", 2)[1]
-
+		message := strings.TrimSuffix(string(buf[:n]), "\n")
 		select {
-		case s.storageQueue <- logMessage:
+		case s.storageQueue <- message:
```

Comment:
>that should probably be a separate issue if it's worthy of further investigation

Sounds good to me. But, since we're trying to send as fast as possible and avoid queue buildup, and we're ok dropping messages (these eventually end up as UDP packages, so I'm assuming...), I bet this is fine as-is.